app/javascript/src/components/form/Dropzone.svelte

```diff
@@ -139,10 +139,10 @@
 
   <small>{ help }</small>
 
-  <label for="" class="dropzone__button button button--secondary mt-1/4">
+  <label class="dropzone__button button button--secondary mt-1/4">
     { button }
 
-    <input type="file" multiple="true" on:change={ changeInput } tabindex="-1" />
+    <input type="file" multiple="true" accept="image/png, image/jpeg" on:change={ changeInput } tabindex="-1" />
```

Comment:
I think it might also need `image/jpg`, if I recall right there are inconsistencies across different browsers if `jpeg` also includes `jpg`

```suggestion
    <input type="file" multiple="true" accept="image/png, image/jpeg, image/jpg" on:change={ changeInput } tabindex="-1" />
```